src/test/java/jewellery/inventory/helper/ProductTestHelper.java

```diff
@@ -0,0 +1,106 @@
+package jewellery.inventory.helper;
+
+import jewellery.inventory.dto.request.ProductRequestDto;
+import jewellery.inventory.dto.request.resource.ResourceQuantityRequestDto;
+import jewellery.inventory.dto.response.ResourcesInUserResponseDto;
+import jewellery.inventory.model.Product;
+import jewellery.inventory.model.ResourceInUser;
+import jewellery.inventory.model.User;
+import jewellery.inventory.model.resource.Resource;
+import jewellery.inventory.model.resource.ResourceInProduct;
+import org.jetbrains.annotations.NotNull;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+
+public class ProductTestHelper {
+
+  public static ResourceInUser createResourceInUser(User user, Resource pearl) {
```

Comment:
I'd say if we are going for get<dto/model> then we should stick to that for all of the dto/model object creation.

Reply:
Done!